### ISSUE TITLE
``whitelist_recursively`` deleted without deprecation

### ISSUE THREAD

**Issue by thomwiggers** [rust-lang/rust-bindgen#2022]
### Bindgen Invocation

```Rust
bindgen::Builder::default()
    .whitelist_recursively()
```

### Actual Results

```
 error[E0599]: no method named `whitelist_recursively` found for struct `bindgen::Builder` in the current scope
Error:   --> oqs-sys/build.rs:23:10
   |
23 |         .whitelist_recursively(false)
   |          ^^^^^^^^^^^^^^^^^^^^^ help: there is an associated function with a similar name: `allowlist_recursively`

```

### Expected Results

One of:
* Deprecation warning, compilation still succeeds, note in the changelog under "deprecated"
* Note in the [changelog](https://github.com/rust-lang/rust-bindgen/blob/master/CHANGELOG.md#removed-1) — this is missing

**Comment by emilio** [comment]
Yeah, this seems like an oversight in #1990, cc @hlopko. Will fix ASAP, thanks for reporting.

**Comment by emilio** [comment]
Thanks for reporting, and thanks @hlopko for #1990 again (sorry, I just noticed the comment above looked a lot like "hey look you broke this", but was intended to be more on the lines of "hey, no need for you to fix this, I'll take care of it" :)).
